src/Services/EkomiServices.php

```diff
@@ -79,6 +79,15 @@ public function sendOrdersData()
             $this->getLogger(__FUNCTION__)->error(self::ERROR_CODE_PLUGIN_DISABLED, $additionalInfo);
         }
 
+        /**
+         * Send data if only terms and conditions are accepted.
+         */
+        if (!$this->configHelper->getTermsAndConditions()) {
+            $additionalInfo = 'terms_and_conditions_accepted:'.$this->configHelper->getTermsAndConditions();
+            $this->getLogger(__FUNCTION__)->error(self::ERROR_CODE_PLUGIN_DISABLED, $additionalInfo);
+            return;
```

Comment:
Is this **return** needed ? I have placed it on my own as i didn't see any check that will stop the code even if the plugin is disabled